src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -27,20 +27,30 @@ g.test('lodMinAndMaxClamp')
 
 g.test('maxAnisotropy')
   .desc('test different maxAnisotropy values and combinations with min/mag/mipmapFilter')
-  .params(
-    params()
-      .combine(poptions('maxAnisotropy', [0, 1, 2, 4, 16, 32, 1024]))
-      .combine(poptions('minFilter', ['nearest', 'linear']))
-      .combine(poptions('magFilter', ['nearest', 'linear']))
-      .combine(poptions('mipmapFilter', ['nearest', 'linear']))
-  )
+  .params([
+    ...poptions('maxAnisotropy', [-1, undefined, 0, 1, 2, 4, 7, 16, 32, 33, 1024]),
```

Comment:
re: some comment somewhere: It's okay not to test other values; we just want to test some values that are allowed by IDL. (Tests for IDL type check rules will be separate from validation tests.) Strictly speaking, we don't even have to test `undefined` here; we should test it when we have an operation test (and it should have the same expectation as `1`, because that is the default value).